books/free-programming-books-ar.md

```diff
@@ -22,7 +22,6 @@
 
 ### Arduino
 
-* [احترف الأردوينو](https://www.ev-center.com/uploads/2/1/2/6/21261678/arduino.pdf) - Working Group&rlm; (PDF)
```

Comment:
this one works for me

Reply:
for some reason, it was not working multiple times. it works for me now.